packages/server-wallet/src/wallet/index.ts

```diff
@@ -104,7 +117,8 @@ export class Wallet implements WalletInterface {
       const {outgoing, channelResult} = await Store.signState(
         channelId,
         {...channelConstants, turnNum: 0, isFinal: false, appData, outcome},
-        tx
+        tx,
+        this && this.options.skipEVMValidation
```

Comment:
It seems like the option is only used in one place, and that place could use the global `config` object.

Is it necessary (preferable?) to pass around a `Wallet` option to the store, modifying the signature of many functions?

Reply:
I think it probably makes sense to just use the global `config` object (I wasn't aware that that was there)

Reply:
resolved 42f6f58